pipelines/snippets.yml

```
@@ -52,94 +52,78 @@ steps:
   submodules: recursive
   persistCredentials: true
 
-- task: PowerShell@2
+- pwsh: |
+    $branchName = "snippet-generation/$env:BUILD_BUILDID" # Match the spec in the GH Action
+    Write-Host "Branch path spec for the pull request will be $branchName"
+    Write-Host "##vso[task.setvariable variable=branchName]$branchName"
   displayName: 'Calculate and set pipeline variables for this job'
-  inputs:
-    targetType: inline
-    script: |
-      $branchName = "snippet-generation/$env:Build_BuildId" # Match the spec in the GH Action
-      Write-Host "Branch path spec for the pull request will be $branchName"
-      Write-Host "##vso[task.setvariable variable=branchName]$branchName"
 
-- task: PowerShell@2
+- pwsh: |
+    Write-Host "The new branch name will be: $env:BRANCHNAME"
+    git checkout -B $env:BRANCHNAME | Write-Host
   displayName: 'Git: branch from master named with the build id: $(Build.BuildId)'
-  inputs:
-    targetType: inline
-    workingDirectory: '$(Build.SourcesDirectory)/microsoft-graph-docs'
-    script: |
-      Write-Host "The new branch name will be: $env:branchName"
-      git checkout -B $env:branchName | Write-Host
+  workingDirectory: '$(Build.SourcesDirectory)/microsoft-graph-docs'
```

I have recently realized that this works too, I think it is easier to read this way.
```suggestion
  workingDirectory: 'microsoft-graph-docs'
```